forge-gui/res/cardsfolder/upcoming/doc_aurlock_grizzled_genius.txt

```diff
@@ -0,0 +1,8 @@
+Name:Doc Aurlock, Grizzled Genius
+ManaCost:G U
+Types:Legendary Creature Bear Druid
+PT:2/3
+S:Mode$ ReduceCost | ValidCard$ Card.wasCastFromYourGraveyard,Card.wasCastFromExile | Type$ Spell | Activator$ You | Amount$ 2 | Description$ Spells you cast from your graveyard or from exile cost {2} less to cast.
+S:Mode$ ReduceCost | ValidSpell$ Static.Plotting | Activator$ You | Amount$ 2 | Description$ Plotting cards from your hand costs {2} less.
```

Comment:
hand check missing

Reply:
fixed - checked it with that Fblthp to make sure!